src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.targets

```diff
@@ -0,0 +1,37 @@
+<!--
+***********************************************************************************************
+Microsoft.Bcl.Compression.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="TraceEventInjectReference" BeforeTargets="ResolveAssemblyReferences" Condition="('$(DesignTimeBuild)' != 'true')">
```

Comment:
Again, I would recommend starting with a completely vanilla file and build up from there.   
I do not know the purpose of TraceEventInjectReference (I think it was there a bug fix for a scenario that we ultimately dropped).    I think it is OK to re-learn why things were done by starting with something simple and understandable, and work up from there.